bundles/org.openhab.binding.bosesoundtouch/src/main/java/org/openhab/binding/bosesoundtouch/internal/XMLResponseHandler.java

```diff
@@ -131,21 +135,18 @@ public void startElement(String uri, String localName, String qName, Attributes
                         state = XMLHandlerState.Unprocessed;
                     }
                 } else {
-                    if (logger.isDebugEnabled()) {
-                        logger.warn("{}: Unhandled XML entity during {}: '{}'", handler.getDeviceName(), curState,
-                                localName);
-                    }
+                    logger.debug("{}: Unhandled XML entity during {}: '{}'", handler.getDeviceName(), curState,
+                            localName);
+
                     state = XMLHandlerState.Unprocessed;
                 }
                 break;
             case MsgHeader:
                 if ("request".equals(localName)) {
                     state = XMLHandlerState.Unprocessed; // TODO implement request id / response tracking...
                 } else {
-                    if (logger.isDebugEnabled()) {
-                        logger.warn("{}: Unhandled XML entity during {}: '{}'", handler.getDeviceName(), curState,
-                                localName);
-                    }
+                    logger.warn("{}: Unhandled XML entity during {}: '{}'", handler.getDeviceName(), curState,
```

Comment:
I don't know how often it happens, but it might spam the log now. It's logged on debug above as well.

```suggestion
                    logger.debug("{}: Unhandled XML entity during {}: '{}'", handler.getDeviceName(), curState,
```

Reply:
done